common/app/views/support/HtmlCleaner.scala

```diff
@@ -682,22 +683,29 @@ case class AtomsCleaner(
         atomId <- Some(bodyElement.attr("data-atom-id"))
         atomType <- Some(bodyElement.attr("data-atom-type"))
       } {
-        findAtom(atomId).fold {
-          atomContainer.remove()
-        } { atomData =>
-          if(mediaWrapper.contains(MediaWrapper.MainMedia)){
-            atomContainer.addClass("element-atom--main-media")
-          }
-          if(atomData.isInstanceOf[MediaAtom]){
-            atomContainer.addClass("element-atom--media")
-          }
+        if (atomType != "audio" || (atomType == "audio" && RenderInArticleAudioAtomSwitch.isSwitchedOn)) {
+          findAtom(atomId).fold {
+            atomContainer.remove()
+          } { atomData =>
+            if(mediaWrapper.contains(MediaWrapper.MainMedia)){
+              atomContainer.addClass("element-atom--main-media")
+            }
+            if(atomData.isInstanceOf[MediaAtom]){
+              atomContainer.addClass("element-atom--media")
+            }
 
-          atomContainer.attr("data-atom-id", atomId)
-          atomContainer.attr("data-atom-type", atomType)
+            atomContainer.attr("data-atom-id", atomId)
+            atomContainer.attr("data-atom-type", atomType)
 
-          val html = views.html.fragments.atoms.atom(atomData, Atoms.articleConfig, shouldFence, amp, mediaWrapper, posterImageOverride).toString()
-          bodyElement.remove()
-          atomContainer.append(html)
+            val html = if(atomData.isInstanceOf[AudioAtom] && isAdFree(request)) {
```

Comment:
In the next update, we can:
- add a flag to [`ArticleConfiguration`](https://github.com/guardian/atom-renderer/blob/master/core/src/main/scala/Configuration.scala#L10-L13)
- pass the `isAdFree(request)` value to it [here](https://github.com/guardian/frontend/blob/0a3c77f4756593e58ca671e5b982281e60f32d5a/common/app/model/content/Atoms.scala#L231-L234)
- use that flag in the `getHTML` template for the audio atom